spec/jobs/solidus_quiet_logistics/outbound/push_shipment_order_document_job_spec.rb

```diff
@@ -12,7 +12,11 @@
     let(:second_shipment_order) { instance_spy(SolidusQuietLogistics::Outbound::Document::ShipmentOrder) }
 
     before do
-      order.shipments << create(:shipment, order: order)
+      if Rails.gem_version > Gem::Version.new(6.1)
```

Comment:
I feel like this will go away once we backport the change @waiting-for-dev did in Solidus. Let's wait for that to see if we can remove the if.

Reply:
This failed locally even after the change, I believe the "If" statement is required here

Reply:
Against which version of Solidus did you see the failure?

Reply:
This fails on Solidus v2.10